src/main/groovy/com/thomsonreuters/lsps/transmart/etl/ClinicalDataProcessor.groovy

```diff
@@ -95,9 +95,14 @@ class ClinicalDataProcessor extends DataProcessor {
                                     def groups = fMappings._DATA.collectEntries{
                                         [(it.DATA_LABEL) : it.COLUMN]
                                     }
+
                                     cat_cd = cat_cd.replaceAll(/\$\$([A-z0-9_\"\s\(\)]+)/) { match, name ->
                                         '$$' + (cols[groups[name]]?.replaceAll(RE_PLUS, '(plus)') ?: 'Not specified')
                                     }
+
+                                    //ignore record without tags value
+                                    if (cat_cd.contains('Not specified'))
+                                        return;
```

Comment:
We should only handle missing tag values in this way, but not all paths which can contain "Not specified" as normal value.